data/batch/batch.pb.go

```diff
@@ -28,7 +28,7 @@ const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 // Transaction holds all the data needed for a value transfer or SC call
 type Batch struct {
-	Data [][]byte `protobuf:"bytes,1,rep,name=Data,proto3" json:"data"`
+	Data [][]byte `protobuf:"bytes,1,rep,name=data,proto3" json:"data"`
```

Comment:
You should revert here, L125, L206, L250

Reply:
done